modules/PropertyManagement/Database/Migrations/2019_07_22_000400_create_Realty_table.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateRealtyTable extends BaseMigration
+{
+    protected $tablename = 'realty';
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create($this->tablename, function (Blueprint $table) {
+            $this->up_table_generic($table);
+
+            $table->integer('node_id')->nullable();
```

Comment:
use unsignedInteger and foreign to tell the database about the relation
and shouldn't a realty always belong to a node?
```suggestion
            $table->unsignedInteger('node_id');
            $table->foreign('node_id')->references('id')->on('node');
```

Reply:
applied